CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
 - `ngl_configure()` does not update the `backend` field from the user
   configuration anymore and the `config` argument is now const. Users must now
   use `ngl_get_backend()` to get the underlying backend information
+- `ngl_scene_init_from_node()` has been replaced with
+  `ngl_scene_init()` with the associated `ngl_scene_params` structure
+- the `ngl_scene` is now private; the parameters can now be obtained using
```

Comment:
"ngl_scene is now private" or "ngl_scene is now a private structure" or "the ngl_scene structure is now private".
its parameters can now be obtained using ...

Reply:
done: added "structure" and used the "its parameters" form